src/main/java/com/cflint/Version.java

```diff
@@ -6,7 +6,12 @@
 
     public static String getVersion() {
         try {
-            return CFLintMain.class.getPackage().getImplementationVersion();
+            String version = CFLintMain.class.getPackage().getImplementationVersion();
+            if (version != null) {
+                return version;
+            } else {
+                return "";
+            }
         } catch (Exception e) {
```

Comment:
I don't think there's any valid case where the code throws an exception. `getImplementationversion()` is just a plain getter. If there's any exception thrown, it'll likely be an error, possibly unrecoverable, that we may want to know about.